server/routes/shared/showReferralView.ts

```diff
@@ -61,6 +61,27 @@ export default class ShowReferralView {
     href: this.presenter.referralOverviewPagePresenter.dashboardURL,
   }
 
+  private readonly insetTextArgs = {
+    html: `${ViewUtils.escape(`You can amend the completion date on this referral.`)}<br>
+          ${ViewUtils.escape(`This will send a notification to the service provider`)}`,
+  }
+
+  get notificationBannerArgs(): NotificationBannerArgs {
+    const html = `<b>Referral changes saved</b>
+          <br>
+          <p>
+             The service provider has been notified of the changes to this referral.
+          </p>
+          <p>
+              <a href= ${this.presenter.closeHref}> close</a>
```

Comment:
nit: should this be `Close` (with no leading space)?

Reply:
also i notice in the designs this is `Dismiss`

Reply:
The design i'm looking at says 'close'